.version

```diff
@@ -1 +1 @@
-2.6.2
```

Comment:
I would just remove this change. We are not really releasing a new version of SAT 2.6, and the versions in the main branch aren't used anyway.

Reply:
Removed